notebooks/official/training/tpuv5e_gemma_peft_finetuning_and_serving.ipynb

```diff
@@ -0,0 +1,1509 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        ! gcloud components update --quiet
</pre><p>No need to update gcloud.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2775/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
gcloud artifacts repositories create didn't recognize the tag --repository-format=docker and it looks like it was due to older components. I can try again without it if you're certain